js/apps/admin-ui/src/user/EditUser.tsx

```diff
@@ -124,7 +124,11 @@ export default function EditUser() {
     try {
       await adminClient.users.update(
         { id: user!.id! },
-        toUserRepresentation(data),
+        {
```

Comment:
The `toUserRepresentation()` function still needs to be called on the data being passed here.